service_catalog/views/request.py

```diff
@@ -72,6 +72,30 @@ def request_comment(request, request_id):
     return render(request, "service_catalog/common/request-comment.html", context)
 
 
+@login_required
+@permission_required_or_403('service_catalog.comment_request', (Request, 'id', 'request_id'))
+def request_comment_edit(request, request_id, comment_id):
+    request_comment = get_object_or_404(RequestMessage, id=comment_id)
+    if request.method == "POST":
+        form = RequestMessageForm(request.POST or None, request.FILES or None, sender=request.user,
```

Comment:
Do not override the sender.
Need to be the same so we still understand who said what.

Reply:
done